ggcore/sdk_messages.py

```diff
@@ -119,3 +119,25 @@ class SavaDatasetResponse(SdkServiceResponse):
     """Define class representing a save dataset api call response."""
     dataset_id: str = None
     save_path: str = None
+
+
+class PropertySource:
+    """Define class representing a source of name/value property pairs"""
```

Comment:
nitpick: periods missing from this and the one below.

Reply:
good catch, fixed